.github/workflows/build.yml

```diff
@@ -64,20 +61,23 @@ jobs:
         run: dotnet restore --locked-mode
         shell: pwsh
 
+      - name: Remove SQL proj
+        run: dotnet sln bitwarden-server.sln remove src/Sql/Sql.sqlproj
```

Comment:
❓ What's the failure? I thought this just warned.

This project type really isn't compatible anymore and Microsoft doesn't seem to want to add its support across the ecosystem outside Windows and VS proper, so I'd support its removal as a project and we manage the files manually. Yes, there are some benefits to its use with compile-like feedback of SQL issues, but if you're not using VS proper (which almost no one here is I hear) then you don't get them.

Reply:
Seems to be needed.